WordPress/src/main/res/layout/quick_start_dialog_fragment.xml

```diff
@@ -1,21 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     style="@style/QuickStartList"
     android:layout_height="match_parent">
 
-    <org.wordpress.android.ui.ActionableEmptyView
-        android:id="@+id/quick_start_complete_view"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:visibility="gone"
-        app:aevImage="@drawable/img_illustration_site_brush_191dp"
-        app:aevSubtitle="@string/quick_start_complete_view_subtitle"
-        app:aevTitle="@string/quick_start_complete_view_title"
```

Comment:
Nitpick: @ashiagr Can we remove the resources which were used here as well?.

Reply:
Removed in 1f073c1, 9daf309